framework/src/media/demux/mpeg2ts/PMTElementary.h

```diff
@@ -0,0 +1,56 @@
+/******************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+#ifndef __PMT_ELEMENTARY_H
+#define __PMT_ELEMENTARY_H
+
+#include "Mpeg2TsTypes.h"
+
+class PMTElementary
+{
+public:
+	enum {
+		STREAM_TYPE_AUDIO_MPEG1  = 0x03,
```

Comment:
please add some description like a below
/** Category value of Streaming type (ISO 13818-1) **/
Also please consider move this to media utils. I think there's chance that used by many places in future..

Reply:
Added descriptions. This is stream type (audio only) enumeration in PMT, it's not exhaustive now, let's keep it internal.